core/templates/pages/story-editor-page/services/story-editor-state.service.ts

```diff
@@ -35,28 +36,40 @@ export class StoryEditorStateService {
   constructor(
     private alertsService: AlertsService,
     private editableStoryBackendApiService: EditableStoryBackendApiService,
+    private loaderService: LoaderService,
     private storyObjectFactory: StoryObjectFactory,
     private undoRedoService: UndoRedoService) {}
 
-  _story: Story = this.storyObjectFactory.createInterstitialStory();
   _storyIsInitialized: boolean = false;
   _storyIsLoading: boolean = false;
   _storyIsBeingSaved: boolean = false;
-  _topicName: string = null;
   _storyIsPublished: boolean = false;
   _skillSummaries: SkillSummaryBackendDict[] = [];
   _expIdsChanged: boolean = false;
   _storyWithUrlFragmentExists: boolean = false;
-  _classroomUrlFragment: string = null;
-  _topicUrlFragment: string = null;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  _story!: Story;
+  _topicName!: string;
+  _classroomUrlFragment!: string;
+  _topicUrlFragment!: string;
```

Comment:
Move these above the variables using default values. And also maybe move the constructor below all the variable definitions.

Reply:
Done.

Reply:
Hey, the lighthouse tests are probably just a flake.

Reply:
Okay, done. I thought it was due to this change, as it failed continuously for 3 times.

Reply:
Hm, once again it failed. I don't think its flake, it is failing because of this change only. Any thoughts?

Reply:
No, it is a flake, I know because we introduced some changes into Lighthouse tests recently.

Reply:
Okay, thanks for confirming.